conditional/blueprints/intro_evals.py

```diff
@@ -33,11 +34,13 @@ def display_intro_evals(internal=False):
     # get user data
     def get_uid_cm_count(member_id):
         return len([a for a in MemberCommitteeAttendance.query.filter(
-            MemberCommitteeAttendance.uid == member_id)])
+            MemberCommitteeAttendance.uid == member_id)
+            if CommitteeMeeting.query.filter(CommitteeMeeting.id == a.meeting_id).approved])
 
```

Comment:
So if I understand this correctly, we go through each item in the MemberCommitteeAttendance query set and for each one query the CommitteeMeeting model to see if it is approved.

Is there a way to just do this with SQL and run a count instead of taking the length of the query set returned? A SQL join on the two tables where the id's match and WHERE approved would be way more efficient